src/components/Activity/ActivityPanel.js

```diff
@@ -35,27 +36,33 @@ const ActivityPanel = ({ open, onBlur, onClearAll }) => {
       {({ x }) => (
         <ActivityFrame
           ref={frameRef}
-          onBlur={_onBlur}
+          onBlur={handleBlur}
           tabIndex="0"
           style={{
             transform: x.interpolate(x => `translate3d(${x}px,0,0)`),
           }}
         >
           <ActivityHeader>
-            <div style={{ display: 'flex', alignItems: 'center' }}>
-              <h1 style={{ marginRight: 10 }}>
-                <Text>Activity</Text>
-              </h1>
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+              `}
+            >
+              <ActivityHeaderTitle>Activity</ActivityHeaderTitle>
             </div>
-            <a onClick={onClearAll}>
-              <Text>Clear All</Text>
-            </a>
+            <Button mode="text" onClick={onClearAll} css="margin-right: -15px">
+              Clear all
+            </Button>
           </ActivityHeader>
-          <ActivityHub
-            activities={activities}
-            keys={activity => activity.transactionHash}
-            clearActivity={clearActivity}
-          />
+          <ActivityContent>
+            <ActivityHub
```

Comment:
Feel free to rename `ActivityHub` so something with more semantic denoting it's a list of sorts.

Reply:
Good idea, I will rename it `ActivityList` :+1: